snuba/datasets/configuration/discover/entities/discover.yaml

```diff
@@ -313,6 +318,7 @@ storages:
               - exception_frames.colno
               - exception_frames.lineno
               - exception_frames.stack_level
+              - exception_main_thread
```

Comment:
I don't know if `DefaultNoneColumnMapper` needs this or not, just adding because `exception_frames` also has it.

Reply:
Yes, this is a list of columns that are in errors but not transactions. We need them listed in Discover.